src/lib/fcitx/candidatelist.h

```diff
@@ -46,12 +46,20 @@ class FCITXCORE_EXPORT CandidateWord {
     bool isPlaceHolder() const;
     bool hasCustomLabel() const;
     const Text &customLabel() const;
+    /**
+     * Return comment corresponding to the candidate.
+     *
+     * @return value of comment.
+     * @since 5.1.9
+     */
+    const Text &comment() const;
```

Comment:
Can we also add a helper function, may be just called `Text textWithComment(std::string separator = " ") const` so we can save some duplicate code in UI.